Story/RainMeadow.StoryHooks.cs

```diff
@@ -1,18 +1,19 @@
 ﻿using UnityEngine;
-using System.Linq;
+using RainMeadow;
+using IL.Menu;
 using System.Collections.Generic;
-using MonoMod.Cil;
-using System;
-using Mono.Cecil.Cil;
+using JollyCoop.JollyHUD;
 using HUD;
-using System.Text.RegularExpressions;
-
+using System.Linq;
+using System.Security.Cryptography;
```

Comment:
This needed?

Reply:
Nope, I will go back through and remove all these auto-added libraries or just wait for @henpemaz to complete his rewrite for the hud